OConnor/resources/views/BloodCalendar.html

```diff
@@ -280,22 +280,27 @@
 	   theWindow.show();      
 
 }
-    
+
+LABKEY.Utils.onReady(function(){
+    document.getElementById('previousButton')['click'] = previous;
```

Comment:
Should this be `onclick`? The changes in this PR are inconsistent.

Reply:
Yes, should be `onclick`. Thanks... I've corrected here and a couple other places.